app/controllers/health_monitor/health_controller.rb

```diff
@@ -10,10 +10,9 @@ class HealthController < ActionController::Base
     def check
       res = HealthMonitor.check(request: request)
 
-      unless HealthMonitor.configuration.environmet_variables.nil?
```

Comment:
HealthMonitor.configuration.environmet_variables can be nil, sure, but it has no meaning even if it is nil we are configuring a default value.

I've changed the styling.